clouda-inc/quickorder
# Fix linting configuration

#### What does this PR do? \*

This PR fixes the linting configuration issue.

#### How to test it? \*

Run `$ yarn install` in both the root directory and the `react/` directory, reload the project files in your IDE and check whether you are seeing linting configuration errors. Further check whether linting errors can be seen in project files.
